here_for_change/municipalities/templates/municipalities/ward_detail.html

```diff
@@ -63,7 +63,7 @@ <h1 class="fvs-250 benfritz w-full text-left text-[48px] leading-[1.2] md:w-auto
   </div>
 </section>
 <section>
-  <div id="map"></div>
+    <iframe class="data-src-wardid data-src-municipalityid" src="{% url 'map' %}" frameborder="0"></iframe>
```

Comment:
@Hazmid do we have to iframe it? Can't we embed it with JS?

Reply:
@paulmwatson Iframe has been removed. map data is now embedded directly.